en/docs/administer/changing-to-oracle.md

```diff
@@ -265,8 +261,7 @@ earlier and run the following scripts in SQL\*Plus:
 2.  Restart the server.
 
     !!! info 
-        You can create database tables automatically **when starting the
-        product for the first time** by using the
+        You can create database tables automatically **when starting the product for the first time** by using the
```

Comment:
Isn't this functionality deprecated now? Please check and update this and similar content.